helm/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+      containers:
+        - name: keycloak
+          image: {{ .Values.image.repo }}keycloak:{{ .Values.keycloakVersion }}
+          imagePullPolicy: Always
+          resources:
+            requests:
+              memory: '512Mi'
+              cpu: '500m'
+            limits:
+              memory: '1Gi'
+              cpu: '1500m'
+          ports:
+            - name: http
+              containerPort: 8080
+          env:
+            - name: KEYCLOAK_USER
```

Comment:
Will this user (with this password) need to be created on the keycloak realm? If so, do we need to add the user to the terraform files so it's managed by terraform?

Reply:
This is how the initial username and password is created in order to access the deployed container. It's what needs to be used for the initial terraform run to create the service account for terraform to use in the pipeline later

Reply:
Ah, ok, that makes sense, no worries on this comment then